src/plugins/crypto.c

```diff
@@ -2273,7 +2372,20 @@ BDCryptoLUKSInfo* bd_crypto_luks_info (const gchar *device, GError **error) {
     info->sector_size = 0;
 #endif
 
+    if (info->version == BD_CRYPTO_LUKS_VERSION_LUKS2) {
+        success = get_subsystem_label (crypt_get_device_name (cd) , &(info->subsystem), &(info->label), error);
+        if (!success) {
+            crypt_free (cd);
+            bd_crypto_luks_info_free (info);
+            return NULL;
+        }
+    } else {
+        info->label = g_strdup ("");
```

Comment:
Does it need to be empty string? Is use of NULL a problem somewhere?

Reply:
We use empty string for the filesystem labels so I just wanted to be consistent. I guess having it always a valid string is easier for people that just want to print/show the label, no need for an extra check to make sure it's not `NULL`/`None`.

Reply:
Okay then, fair enough.